core/templates/dev/head/components/forms/schema_editors/SchemaBasedHtmlEditorDirective.js

```diff
@@ -23,11 +23,29 @@ oppia.directive('schemaBasedHtmlEditor', [
         localValue: '=',
         isDisabled: '&',
         labelForFocusTarget: '&',
-        uiConfig: '&'
+        uiConfig: '&',
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/forms/schema_editors/' +
         'schema_based_html_editor_directive.html'),
-      restrict: 'E'
+      restrict: 'E',
+      link: function($scope, element, $attrs) {
+        var observer = new MutationObserver(function(mutations) {
+          if (mutations[mutations.length - 1].addedNodes[0]) {
+            if (mutations[mutations.length - 1]
+              .addedNodes[0].nodeName === 'IMG'){
+              var imageRteComeponent =
```

Comment:
Minor typo here, should be `imageRteComponent`

Also, here and below better to split newline in a better place (for example, at the opening `[`).

Reply:
Done.